src/rpc/blockchain.cpp

```diff
@@ -448,14 +449,14 @@ UniValue getblockdeltas(const UniValue& params, bool fHelp)
             "      ]\n"
             "    }, ...\n"
             "  ],\n"
-            "  \"time\": n,\n"
-            "  \"mediantime\": n,\n"
-            "  \"nonce\": \"hexstring\",\n"
-            "  \"bits\": \"hexstring\",\n"
-            "  \"difficulty\": ,\n"
-            "  \"chainwork\": \"hexstring\",\n"
-            "  \"previousblockhash\": \"hash\",\n"
-            "  \"nextblockhash\": \"hash\"\n"
+            "  \"time\" : n,                  (numeric) The block version\n"
+            "  \"mediantime\": n,             (numeric) The most recent blocks' ave time\n"
```

Comment:
```suggestion
            "  \"mediantime\": n,             (numeric) The 11 most recent blocks' median time\n"
```

Reply:
I'll do this in that follow-up cleanup PR, since all three of you have already ACKed.